model/rag/chat.go

```diff
@@ -128,15 +127,27 @@ func Query(inst *instance.Instance, logger logger.Logger, query QueryMessage) er
 	if err != nil {
 		return err
 	}
-	myself, _ := contact.GetMyself(inst)
-	relatives, _ := getRelatives(inst, myself)
+
+	type RAGMessage struct {
+		Role    string `json:"role"`
+		Content string `json:"content"`
+	}
+
+	chat_history := make([]RAGMessage, 0, len(chat.Messages))
+	for _, msg := range chat.Messages {
+		chat_history = append(chat_history, RAGMessage{
+			Role:    msg.Role,
+			Content: msg.Content,
+		})
+	}
+
 	payload := map[string]interface{}{
-		"locale":       inst.Locale,
-		"messages":     chat.Messages,
-		"myself":       myself,
-		"relatives":    relatives,
-		"stream":       true,
-		"with_sources": true,
+		"model":       fmt.Sprintf("ragondin-%s", inst.Domain),
+		"messages":    chat_history,
+		"stream":      true,
+		"temperature": 0.3,
+		"top_p":       1,
+		"logprobs":    0,
```

Comment:
For those 3 parameters, it could be nice to extract the values into constants with a comment that explains how the values were chosen.

Reply:
Right, it's done